.circleci/tests_unit/test_parsing_integration.py

```diff
@@ -0,0 +1,279 @@
+"""
+Integration tests for parsing real YAML test definition files.
```

Comment:
I don't like having filenames all over the place. This should be automatic by directory browsing with python pathlib.